sw/device/lib/testing/spi_flash_testutils.c

```diff
@@ -0,0 +1,207 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/spi_flash_testutils.h"
+
+#include "sw/device/lib/base/macros.h"
+#include "sw/device/lib/dif/dif_spi_host.h"
+#include "sw/device/lib/testing/spi_device_testutils.h"
+#include "sw/device/lib/testing/test_framework/check.h"
+
+void spi_flash_testutils_read_id(dif_spi_host_t *spih,
+                                 spi_flash_testutils_jedec_id_t *id) {
+  CHECK(spih);
```

Comment:
Be explicit: `CHECK(spih != NULL);`

Reply:
Oops, went back to kernel style. Though... considering that text is also the explainer error message for this macro, being explicit would be preferred there too, heh.